src/System.Net.Sockets/tests/FunctionalTests/UnixDomainSocketTest.cs

```diff
@@ -202,6 +202,67 @@ public void Socket_SendReceive_Success()
         }
 
         [OuterLoop] // TODO: Issue #11345
+        [Theory]
+        [InlineData(5000, 1, 1, false)]
+        [InlineData(5000, 1, 1, true)]
```

Comment:
We can add some.  These are the buffer sizes that are causing grief in the corresponding test that hangs in System.IO.Pipes.